elastic/kibana
# [ML] Support trained model aliases

## Summary

Resolves https://github.com/elastic/kibana/issues/93392

- Fetches ingest pipelines associated with model's aliases 
![image](https://user-images.githubusercontent.com/5236598/116985393-aebf2380-accc-11eb-86b1-ce60d94a106a.png)

- Adds an extra panel to the expandable row to show models metadata 
![image](https://user-images.githubusercontent.com/5236598/116985433-c0a0c680-accc-11eb-8d10-38ecd78e272d.png)


### How to test 

1. Create a trained model using API, or use the one created by the DFA job. 
2. Create an alias for this model making an API call in Dev Tool, check API docs how to write a call https://www.elastic.co/guide/en/elasticsearch/reference/master/put-trained-models-aliases.html
3. Set up a pipeline using the alias you created as a `model_id`
4. You should see this pipeline in the expanded row of the model


### Checklist
